db/seeds.rb

```diff
@@ -6,3 +6,16 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
+if ENV['USE_DEMO_DATA'] || Settings.use_demo_data || Rails.env.test?
```

Comment:
It's difficult to remove the demodata flow from MessagesController if we still want the demodata functionality on the playground page [we are relying on the user being familiar with what the page is supposed to do - or maybe put in some safety such as if repeat_demo >0 and the data does not contain {..} ignore this repeat_demo parameter or similiar..], or maybe we create a new api/demoplayground web page or something [but thats maybe duplicating code/effort]. what do you think?

-just creating a rake task now as from the comments and moving the template seed data files.